chart/gatekeeper-operator/templates/gatekeeper.yaml

```diff
@@ -463,7 +463,7 @@ spec:
           httpGet:
             path: /readyz
             port: 9090
-        resources:
+        resources: 
```

Comment:
trailing space ?

Reply:
This is likely output from `make manifests`. Is a trailing space breaking in some way?

Reply:
nah, just nice to get rid of them all :D

Reply:
ack, I think these should be kept for now. The only way to get rid of them would be to modify

https://github.com/open-policy-agent/gatekeeper/blob/0f343281091853fa20c3ff7149166826224b9408/chart/gatekeeper-operator/generate_helm_template.sh#L6

and the other sed statements to also replace spaces, which seems like it runs the risk of making that code more brittle.

Reply:
yeah, no big deal :)